backend/django/core/migrations/0066_alter_externaldatabase_project.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 3.2.9 on 2022-01-30 16:05
+
+import django.db.models.deletion
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("core", "0065_alter_externaldatabase_project"),
+    ]
+
```

Comment:
I'll admit that I'm not super familiar with the Migration piece and wasn't entirely sure if there was anything specific I should look for here when reviewing. If so, feel free to let me know

Reply:
The migrations are automatically generated. So I guess the main thing to review is really the models.py file and the new tables/fields defined in it